DiverseMarket.UI/Pages/Company/Product/AddNewProductPage.Designer.cs

```diff
@@ -0,0 +1,387 @@
+        //private Label categoryWarningLabel;
+        //private Label quantityWarningLabel;
+        //private Label priceWarningLabel;
+        #endregion
+
+        #region RoundedTextBoxes
+        private RoundedTextBox nameTextBox;
+        private RoundedTextBox descriptionTextBox;
+        private RoundedTextBox categoryTextBox;
+        private RoundedTextBox quantityTextBox;
+        private RoundedTextBox priceTextBox;
+        private ComboBox categoryComboBox = new ComboBox();
+
+        #endregion
+
+        #region Buttons
+        private Button homepageButton;
+        private Button returnButton;
+        private Button addNewOfferButton;
+        #endregion
+
+        #region Dispose
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing && (components != null))
+            {
+                components.Dispose();
+            }
+            base.Dispose(disposing);
+        }
+
+        #endregion
+
+        #region Init Components
+        private void InitializeComponent(long userId)
+        {
+            this._userId = userId;
+            AutoScaleMode = AutoScaleMode.Font;
+            ClientSize = new Size(1280, 832);
+            StartPosition = FormStartPosition.CenterScreen;
+            Text = "DiverseMarket";
+            this.BackColor = Colors.MainBackgroundColor;
+            FormClosed += _FormClosed;
+            InitScreen();
+        }
+
+        private void InitScreen()
+        {
+            InitLogo();
+            InitTextBoxes();
+            InitLabel();
+            InitButtons();
+        }
+
+        #endregion
+
+        #region Text Boxes
+        private void InitTextBoxes()
+        {
+            int y = 200;
+            int spacing = 50;
+
+            nameTextBox = new RoundedTextBox(string.Empty, 572, 40);
+            nameTextBox.Location = new Point(354, y);
+            nameTextBox.TextBox.Font = new Font("Ubuntu", 10);
+            this.Controls.Add(nameTextBox);
+
+            descriptionTextBox = new RoundedTextBox(string.Empty, 572, 40);
+            descriptionTextBox.Location = new Point(354, nameTextBox.Bottom + spacing);
+            descriptionTextBox.TextBox.Font = new Font("Ubuntu", 10);
+            this.Controls.Add(descriptionTextBox);
+
+
+
+            /*categoryComboBox.Font = new Font("Ubuntu", 10);
+            categoryComboBox.Size = new Size(572, 40);
+            categoryComboBox.Location = new Point(354, descriptionTextBox.Bottom + spacing);
+            List<string> productsFromService = ProductService.GetAllProductCategories();
+            categoryComboBox.DataSource = productsFromService;
+
+            quantityTextBox = new RoundedTextBox(string.Empty, 572, 40);
+            quantityTextBox.Location = new Point(354, categoryComboBox.Bottom + spacing);
+            quantityTextBox.TextBox.Font = new Font("Ubuntu", 10);
+            this.Controls.Add(quantityTextBox);*/
+
```

Comment:
```suggestion
            categoryComboBox.Font = new Font("Ubuntu", 10);
            categoryComboBox.Size = new Size(572, 40);
            categoryComboBox.Location = new Point(354, descriptionTextBox.Bottom + spacing);
            List<string> productsFromService = ProductService.GetAllProductCategories();
            categoryComboBox.DataSource = productsFromService;
            this.Controls.Add(categoryComboBox);

```

Faltou essa última linha pra adicionar aos Controls o combobox.